arm-oraclelinux-wls-cluster/src/main/arm/nestedtemplates/aadNestedTemplate.json

```diff
@@ -59,6 +59,22 @@
                 "description": "Location for all resources."
             }
         },
+        "managedServerPrefix": {
+            "type": "string",
+            "defaultValue": "msp",
+            "metadata": {
+                "description": "Provide managed server prefix name"
+            }
+        },
+        "numberOfInstances": {
+            "type": "int",
+            "defaultValue": 2,
+            "minValue": 2,
+            "maxValue": 5,
+            "metadata": {
+                "description": "Number of VMs to deploy, limit 5 since this sample is using a single storage account"
+            }
```

Comment:
Add managed parameters, used to update aad ip, import cer and enable tlsv1.2 in managed vms with vm extension.